assets/facets.js

```diff
@@ -164,6 +165,9 @@ class FacetFiltersForm extends HTMLElement {
       {
         section: document.getElementById('product-grid').dataset.id,
       },
+      {
+        section: document.getElementById('main-collection-filters').dataset.id,
+      },
```

Comment:
One thing I noticed while testing is that with those changes we're getting two network requests while if I look at Dawn untouched I'm getting one request when filtering 🤔 One for the filter and one for the product grid. 

They're both lighter individually but since there are 2, I end up with 7kb x 2. While on untouched Dawn I get 11kb and only one request. 

Unless I'm missing something while testing.

Reply:
That seems expected with the way the script currently grabs one section per request @ https://github.com/Shopify/dawn/blob/02aaed23526b99fd6beb851cd3ff1b751960acd3/assets/facets.js#L46

Reply:
Then I'm not sure it's worth decoupling the filters from the product grid 🤔 
Is there a benefit to having 2 smaller fetch vs 1 that's bigger but equal or lower than the two individual ones ?